load-testing/post-datadog-metrics.js

```diff
@@ -0,0 +1,118 @@
+const dgram = require('dgram');
```

Comment:
Any reason we don't just use the Datadog plugin: https://www.artillery.io/docs/reference/extensions/publish-metrics#datadog?

Reply:
it wasn't working

Reply:
Would like more info on why it wasn't working because it really seems like it should given the level of support indicated in the Artillery docs.

Reply:
Did try debug- 
DEBUG=plugin:publish-metrics:datadog-statsd artillery run my-script.yaml
Data was not showing in Datadog, no apparent error.

Did try installing local datadog agent too. No help!
Datadog API worked right away - with same key and secret

We can try again in future! We have a workaround for now
Do report metrics to datadog
```
yarn test:load-dd
```

Reply:
Using work around for now